components/EmotionDetails.js

```diff
@@ -48,21 +49,27 @@ export default function EmotionDetails({ object, onDeleteEmotion }) {
 }
 
 // Styled Components //
-const StyledLink = styled(Link)`
+const StyledBackLink = styled(Link)`
   font-size: 24px;
   margin-left: 10px;
   text-decoration: none;
   color: #000;
 `;
 
-const StyledContainer = styled.div`
-  font-size: 1rem;
-  margin: 16px 8px;
-  padding: 10px;
-  background-color: #f9f9f9;
-  border: 1px solid #d3d3d3;
-  border-radius: 8px;
-  list-style: none;
+const StyledEditLink = styled(Link)`
+  float: left;
+  margin-right: 20px;
+  margin: 10px;
```

Comment:
We were advised to only use `margin` in emergencies and to work with `padding` wherever possible, because `margin` can cause problems. I would also recommend the `margin` shorthand property if you have several defined margin values

Reply:
Good point! We will and already take care within our styling user story!